spec/services/course/duplication_service_spec.rb

```diff
@@ -38,6 +38,11 @@
           expect(new_course.end_at).to be_within(1.second).of course.end_at + 1.day
         end
 
+        it 'sets new course registration key to nil' do
```

Comment:
Test that a course with registration key can be duplicated?  It's not that helpful to just test what the code exactly does.